gradle/versioning.gradle

```diff
@@ -0,0 +1 @@
+version = '2.9.0-SNAPSHOT'
```

Comment:
Can we put the intellij version here, too? And the versions of the used gradle plugins?